autogpt_platform/docker-compose.platform.yml

```diff
@@ -66,6 +66,7 @@ services:
       - ENABLE_AUTH=true
       - PYRO_HOST=0.0.0.0
       - EXECUTIONMANAGER_HOST=executor
+      - BACKEND_CORS_ALLOW_ORIGINS=["http://localhost:3000"]
```

Comment:
Is this loaded correctly? In any case it's not a defined or documented behavior.

Reply:
yeah, this is what works - as you pointed out above. Pydantic handles parsing it into a list